main.go

```diff
@@ -71,16 +72,22 @@ func main() {
 	networkHandler := network.NewHandler(driver.NewNetworkDriver(client))
 	ipamHandler := ipam.NewHandler(driver.NewIpamDriver(client))
 
+	rootGid, err := lookupGidByName("root")
+	if err != nil {
+		log.Infoln("calico-libnetwork-plugin has stopped working.")
+		log.Fatalln(err)
+		return
+	}
 	go func(c chan error) {
 		log.Infoln("calico-net has started.")
-		err := networkHandler.ServeUnix("root", networkPluginName)
+		err := networkHandler.ServeUnix(networkPluginName, rootGid)
```

Comment:
Yes, I think it would be fine to just use a hardcoded "0" here (and below), maybe pulled out as a constant with a small comment.